lib/assets/javascripts/turbograft/turbolinks.coffee

```diff
@@ -118,15 +180,14 @@ class window.Turbolinks
     triggerEvent 'page:receive'
     options.updatePushState ?= true
 
-    if doc = processResponse(xhr, options.partialReplace)
-      reflectNewUrl url if options.updatePushState
-      nodes = changePage(extractTitleAndBody(doc)..., options)
-      reflectRedirectedUrl(xhr) if options.updatePushState
+    doc = processResponse(xhr, options.partialReplace)
```

Comment:
that seems a bit hacky and I don't see any reason why we'd do that if we we could do it the proper way, but I don't feel that strongly